package/boot/arm-trusted-firmware-mvebu/patches/010-fix-compile-wrapper.patch

```diff
@@ -0,0 +1,13 @@
+--- a/scripts/ci/compile.sh
++++ b/scripts/ci/compile.sh
+@@ -14,8 +14,8 @@
+ 
+ 
+ ## =v=v=v=v=v=v=v=v=v=v=v CI WRAPPER - Do not Modify! v=v=v=v=v=v=v=v=v=v=v= ##
+-set -euo pipefail
+-shopt -s extglob
++#set -euo pipefail
```

Comment:
Why is this modification needed?

Reply:
Marvell has some internal CI system which uses this script for the build, but these options are not required for the OpenWRT and might cause build to fail.

Reply:
> build to fail

How exactly? I simply can't find it referenced in the Makefile.

Reply:
sorry, you're right. We switched from the Marvell's ATF package to mainline one so this is not needed anymore. Updated PR to remove the patch.